src/pages/products/Filters.jsx

```diff
@@ -1,93 +1,250 @@
+                  payload: HIGH_LOW_PRICE,
+                })
+              }
+            />
+
+            <FilterInput
+              // key={_id}
+              title="Rating - Low to High"
+              inputType="radio"
+              className="radio"
+              name={ACTION_TYPE.SORT_BY}
+              checked={filterState.sortBy === LOW_HIGH_RATING}
+              // value={LOW_HIGH_PRICE}
+              changeHandler={(e) =>
+                dispatch({
+                  type: ACTION_TYPE.SORT_BY,
+                  payload: LOW_HIGH_RATING,
+                })
+              }
+            />
+
+            <FilterInput
+              // key={_id}
+              title="Rating - High to Low"
+              inputType="radio"
+              className="radio"
+              name={ACTION_TYPE.SORT_BY}
+              checked={filterState.sortBy === HIGH_LOW_RATING}
+              // value={LOW_HIGH_PRICE}
+              changeHandler={(e) =>
+                dispatch({
+                  type: ACTION_TYPE.SORT_BY,
+                  payload: HIGH_LOW_RATING,
+                })
+              }
+            />
+          </fieldset>
+
+          <fieldset className="my-2">
+            <legend className="fw-bold">Product Type</legend>
+
+            {types.map(({ _id, productType }) => {
+              return (
+                <FilterInput
+                  key={_id}
+                  title={productType}
+                  inputType="checkbox"
+                  // name={name}
+                  checked={filterState.productTypes.some(
+                    (item) => item === productType.toLowerCase()
+                  )}
+                  value={productType}
+                  changeHandler={(e) =>
+                    dispatch({
+                      type: ACTION_TYPE.PRODUCT_TYPE,
+                      payload: e.target.value.toLowerCase(),
+                    })
+                  }
+                />
+              );
+            })}
+          </fieldset>
+
+          <fieldset className="my-2">
+            <legend className="fw-bold">Categories</legend>
+
+            {categories.map(({ _id, categoryName }) => {
+              return (
+                <FilterInput
+                  key={_id}
+                  title={categoryName}
+                  inputType="checkbox"
+                  checked={filterState.categories.some(
+                    (item) => item === categoryName.toLowerCase()
+                  )}
+                  value={categoryName}
+                  changeHandler={(e) =>
+                    dispatch({
+                      type: ACTION_TYPE.CATEGORIES,
+                      payload: e.target.value.toLowerCase(),
+                    })
+                  }
+                />
+              );
+            })}
+          </fieldset>
+
+          <fieldset className="my-2">
+            <legend className="fw-bold">Rating</legend>
+
+            <FilterInput
```

Comment:
You can take an array [1,2,3,4,5] and write the ratings logic inside it.
This will help reduce the code.

Reply:
Done